src/client/reducers/index.js

```diff
@@ -1,12 +1,16 @@
 import { combineReducers } from 'redux';
 
 import { reducer as chartTimePeriodIndex } from './chartTimePeriodIndex';
+import { reducer as isFetching } from './isFetching';
 import { reducer as search } from './search';
 import { reducer as selectedStock } from './selectedStock';
 import { reducer as stocks } from './stocks';
+import { reducer as searchError } from './searchError';
 
 export const reducer = combineReducers({
   chartTimePeriodIndex,
+  isFetching,
+  searchError,
```

Comment:
Ok I do think `searchError` should be inside of state.search, instead of on the same level

Reply:
state.search.error instead of state.search.searchError